content/src/quick-action-license-reinstatements/hvacr-license-reinstatement.md

```diff
@@ -10,26 +10,28 @@ summaryDescriptionMd: "\n"
 callToActionText: " Reinstate My  HVACR License "
 callToActionLink: https://www.njconsumeraffairs.gov/hvacr/Pages/applications.aspx
 ---
+
 If your Heating, Ventilating, Air Conditioning, and Refrigeration (HVACR) license has expired, you can apply for reinstatement.
-___
+
+---
 
 ### Reinstatement Requirements
 
-*  `EIN|EIN` 
-* Social Security number (SSN)
-* Suspended license number
-* List of licenses or certificates (past and present) 
-* Certification of employment, listing all jobs held during suspension
-* Certificate of completion for continuing education 
-*  `Surety bond|surety-bond`
```

Comment:
nothing changed

Reply:
Formatting. It's a disagreement between Prettier and Decap about how to do lists.